src/System.Reflection.Metadata/src/System/Reflection/Metadata/Handles.cs

```diff
@@ -652,6 +652,18 @@ public override int GetHashCode()
         {
             return left._rowId != right._rowId;
         }
+
+        /// <summary>
+        /// Returns a handle to <see cref="MethodDebugInformation"/> corresponding to this handle.
+        /// </summary>
+        /// <remarks>
+        /// The resulting handle is only valid within the context of a <see cref="MetadataReader"/> open on the Portable PDB blob,
+        /// which in case of standalone PDB file is a different reader than the one containing this method definition.
+        /// </remarks>
+        public MethodDebugInformationHandle ToDebugInformationHandle()
```

Comment:
Is there any real reason to have it on Handle then? I would prefer to not have the dependency in Handle and keep the dependency from MethodDebugInformationHandle to Handle instead of the reverse.